target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -164,6 +164,11 @@ case "$FIRMWARE" in
 		ath10kcal_extract "art" 0x5000 0x844
 		ath10kcal_patch_mac $(macaddr_add $(mtd_get_mac_binary romfile 0xf100) +2)
 		;;
+	tplink,archer-d7-v1|\
+	tplink,archer-d7b-v1)
+		ath10kcal_extract "art" 20480 2116
+		ath10kcal_patch_mac $(macaddr_add $(mtd_get_mac_binary romfs 0xf100) +2)
```

Comment:
Something speaking against naming the partition "romfile" and merging with d50?
Or rename d50 to romfs ...

Reply:
I was just using the original OEM partitions names... Should I rename it anyway?

Reply:
Well, it's only those two entries at the moment (with romf* or 0xf100). One could wait for other similar devices and then use the more abundant (i.e. change nothing now).
Or one could just decide for one name right now and set the standard with that.
I'd say just choose what you like better ...
(But convert to hexadecimal if you decide two have d7 as separate entry).

Reply:
> (But convert to hexadecimal if you decide two have d7 as separate entry).

I was just doing that conversion ;)